Cargo.toml

```diff
@@ -6,3 +6,6 @@ members = [
   "deno_typescript",
   "test_plugin"
 ]
+exclude = [
+  "std/hash/_wasm"
+]
```

Comment:
Why exclude it? Maybe it's better to add this to the workspace, so we can ensure it is always building?

Reply:
I thought we don't need to build wasm everytime we build deno.
Do you think we have to?